package_manifest.yaml

```diff
@@ -15,21 +15,10 @@ packages:
     build_package_use_koji_build: true
```

Comment:
Is this also still relevant? I think this was made redundant when we switched to copr.